IaC/report-uptime.yml

```diff
@@ -0,0 +1,36 @@
+- job:
+    name: uptime-report
+    project-type: pipeline
+    description: |
+        report uptime of jenkins master
+        Do not edit this job through the web!
+    disabled: false
+    concurrent: false
+    properties:
+    - build-discarder:
+        days-to-keep: -1
+        num-to-keep: 10
+        artifact-days-to-keep: 0
+        artifact-num-to-keep: 0
+    triggers:
+    - timed: '*/5 * * * *'
+    dsl: |
+        node('master') {
+          sh """
+            UNIT=\$(uptime -p | awk '{print \$3}')
```

Comment:
why do we need UNIT?
you can just do the following: 
```
if [[ \$(awk '{print \$1}' /proc/uptime) / 60 | bc) -lt 15 ]]; then
    echo alert > status
else
    echo ok > status
fi
```

Reply:
I was about to push it
Build: https://rel.cd.percona.com/job/uptime-report/692/